Feature v1.0.0

simandebvu opened the pull request:
## [1.0.0] - 2020-08-19 🌤️ 
### Added 💻 
- Intuitive GUI
- GIF in search results
- Validation on empty search text, too few characters and too many characters
- Link to developers repository

simandebvu committed:
Run Initial Setup

simandebvu committed:
Create navbar module

simandebvu committed:
Finish Search

simandebvu committed:
Style error page

simandebvu committed:
Update readme, make units variable

simandebvu committed:
Update search parameter

simandebvu committed:
Parameterise search units

simandebvu committed:
Add Live Demo Link

Ispirett reviewed (changes requested):
## Status Changes required

Great work so far @simandebvu  you are doing well. 👍 

Some things to take care of.

- [x] I tried a couple of cities from the states and the do not work.
- [x] I suggest displaying the name of the weather API you leveraged to build your app.

simandebvu committed:
Use https in async calls

Fixing this problem

`This request has been blocked; the content must be served over HTTPS.`

simandebvu committed:
Add info about the API Sources

simandebvu committed:
Update Changelog

simandebvu committed:
Error handle giphy API Calls

simandebvu commented:
### Fixed 🧑‍🔧 

- Use HTTPS for API Calls
` The API was working correctly, the browser was also rightfully throwing a mixed content error - issue has since been resolved and code was refactored. Thank you for pointing that out.`
- Separate logic for loading from giphy

simandebvu committed:
Update min length

Ispirett reviewed (approved):
## Status Approved 👍 

Great job with the changes made 👍 you can move on now.
